VideoWeb/VideoWeb/ClientApp/src/app/services/media-stream.service.spec.ts

```diff
@@ -157,7 +157,7 @@ describe('MediaStreamService', () => {
             // Assert
             expect(mediaDevicesSpy.getUserMedia).toHaveBeenCalledWith(cameraConstraintBuilder(cameraDevice));
 
-            expect(resultantStream).toBe(filterStream);
+            expect(resultantStream).toBe(expectedStream);
```

Comment:
Has this changed because of the additional condition in the service?

Reply:
yes